src/Http/Http.Features/src/SameSiteMode.cs

```diff
@@ -1,14 +1,20 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 namespace Microsoft.AspNetCore.Http
 {
-    // RFC Draft: https://tools.ietf.org/html/draft-ietf-httpbis-cookie-same-site-00
+    /// <summary>
+    /// Determines whether to send a cookies on "same-site" or "cross-site" requests.
```

Comment:
This phrasing is a bit misleading. This enum is used when generating response cookies and controls adding said fields to the cookie. It does not control the servers behavior regarding cookies. The client is the one that reacts to the values and decides if it should return the cookie on future requests.

```suggestion
    /// Used to set the SameSite field on response cookies to indicate if those cookies should be included by the client on future "same-site" or "cross-site" requests.
```